enhancements/console/quick-starts.md

```diff
@@ -151,6 +155,30 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `label` contains a `language` field to specify what
+language the Quick Start is in. CRDs without this label will be treated
+as English CRDs. OpenShift can display the correct
+Quick Starts for each language by comparing the value of the
+`language` label to the currently displayed language in OpenShift.
+(The display language is determined by `localStorage.getItem('bridge/language')`
```

Comment:
We might move this to the user settings config map. I would leave this detail out and instead link to the i18n enhancement for more information.